AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1,3 +1,10 @@
+!
+! This is section contains the list of the rules that is used for CSS tweaks. For instance, we unblock right context menu, disabled selection of the text - things that are not related to ads.
```

Comment:
`CSS tweaks` - not only.
`things that are not related to ads` redundant and obviously in this filter)